src/lib/Components/Consignments/Screens/Metadata.tsx

```diff
@@ -48,7 +48,7 @@ interface State extends ConsignmentMetadata {
 export default class Metadata extends React.Component<Props, State> {
   constructor(props) {
     super(props)
-    this.state = props
+    this.state = props.metadata || {}
```

Comment:
https://twitter.com/orta/status/908405026149797890

Reply:
This wasn't caught by TypeScript, which was something I noted - the `props` that comes in through the constructor is an `any`, which _I think_ is wrong.